core/templates/dev/head/components/side_navigation_bar/side_navigation_bar_directive.html

```diff
@@ -77,6 +77,9 @@
       height: 56px;
       padding-top: 6px;
     }
+    .oppia-sidebar-menu-icon.fa {
```

Comment:
Then, if it's not a material icon, why does the .material-icons class show up in that element?